src/drawing/DrawingCreator.scss

```diff
@@ -1,12 +1,3 @@
-
-.ba-DrawingCreator {
-    cursor: url($ba-DrawingCreator-cursor) 7 25, crosshair; /* Legacy */
```

Comment:
Should we retain the fallback cursor of `crosshair` for browsers that don't support custom SVG cursors?